tasks/extensions/postgis.yml

```diff
@@ -12,3 +12,19 @@
     - "postgresql-{{postgresql_version}}-postgis-scripts"
   notify:
     - restart postgresql
+  when: ansible_os_family == "Debian"
+
+
+- name: PostgreSQL | Extensions | Make sure the postgis extensions are installed | RedHat
+  yum:
+    name: "{{item}}"
+    state: present
+  with_items:
+    - "postgis2_{{postgresql_version_terse}}"
+    - "postgis2_{{postgresql_version_terse}}-client"
+    - "postgis2_{{postgresql_version_terse}}-devel"
```

Comment:
I agree that the dev packages are not required for production use. On the other hand the role already has the option to install dev packages (headers) if one choses so. So my thought was that if you install the PostgreSQL dev packages and chose to select PostGIS there is a good chance that the user wants the PostGIS dev package, too. Having to install it manually defeats the purpose of automation. For example, I am using the role together with Vagrant to create a PostGIS development server.